src/main/java/com/sintern/api/ApplicationController.java

```diff
@@ -0,0 +1,28 @@
+package com.sintern.api;
+
+import com.sintern.service.ApplicationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@CrossOrigin
+@RestController
+@RequestMapping("/applications")
+public class ApplicationController {
+    private final ApplicationService applicationService;
+
+    @Autowired
+    public ApplicationController(ApplicationService applicationService) {
+        this.applicationService = applicationService;
+    }
+
+    @PostMapping("/add")
```

Comment:
Since we're using a **POST HTTP** method, we don't have to specify `/add`, because **POST** usually means **ADD** (**PUT** for **UPDATE**, **DELETE** for **DELETE** etc.). Please delete `/add`.

Reply:
Solved, please see the last commit!